io-engine/src/bdev/nvme.rs

```diff
@@ -159,7 +156,7 @@ const MAX_NAMESPACES: usize = 1;
 struct NvmeCreateContext {
     trid: spdk_nvme_transport_id,
     names: [*const c_char; MAX_NAMESPACES],
-    prchk_flags: u32,
+    // prchk_flags: u32,
```

Comment:
why is this bit commented out now?

Reply:
"prchk_flags" were here to be passed to bdev_nvme_create(), but we never assigned anything meaningful.
In SPDK 22.05, "prchk_flags" argument is no longer passed directly to bdev_nvme_create(). It is part of "nvme_ctrlr_opts" arg instead, which we do not use.
I'll remove the commented code. If we need this parameter in future, we'll add it again somewhere.

Reply:
fixed